app/models/concerns/decidim/vocdoni/authorization_override.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Vocdoni
+    module AuthorizationOverride
+      extend ActiveSupport::Concern
+
+      included do
+        has_one :vocdoni_authorizations_data,
+                class_name: "Decidim::Vocdoni::AuthorizationsData",
+                dependent: :destroy
```

Comment:
If an authorization is detroyed, this will destroy this entry. Shouldn't we update the vocdoni census again?